resources/todomvc/architecture-examples/vue/src/components/TodosComponent.vue

```diff
@@ -23,7 +23,9 @@ function uuid() {
     let uuid = "";
     for (let i = 0; i < 32; i++) {
         let random = (Math.random() * 16) | 0;
-        if (i === 8 || i === 12 || i === 16 || i === 20) uuid += "-";
+        // prettier-ignore
```

Comment:
I have to look into the vue built to see why this doesn't format, but for consistency I overwrote it manually.